README.md

```diff
@@ -1,29 +1,4 @@
-**This FAQ may be outdated. Find our most up-to-date documentation [here](https://metamask.github.io/metamask-docs/)**
+# This FAQ has been archived.
 
-# MetaMask FAQ
+## Find our most up-to-date documentation [here](https://metamask.github.io/metamask-docs/)
```

Comment:
We should probably link to the live site:
```suggestion
## Find our most up-to-date documentation [here](https://docs.metamask.io)
```

Reply:
Ah right, good catch!